apps/web/src/views/LimitOrders/index.tsx

```diff
@@ -408,6 +408,9 @@ const LimitOrders = () => {
                       showCommonBases
                       commonBasesType={CommonBasesType.SWAP_LIMITORDER}
                     />
+                    <Box>
```

Comment:
Add an array to check that is network supported?
The red alarm api only work in bsc